doc/python/time-series.md

```diff
@@ -169,7 +169,17 @@ df = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/finan
 df = df.loc[(df["Date"] >= "2016-07-01") & (df["Date"] <= "2016-12-01")]
 
 fig = px.line(df, x='Date', y='AAPL.High')
-fig.update_xaxes(ticks= "outside", ticklabelmode= "period", tickcolor= "black", tickwidth=2, ticklen=10, minor=dict(ticks="outside", dtick=7*24*3.6e6,  tick0="2016-07-04", griddash='dot', gridcolor='pink'))
+fig.update_xaxes(ticks= "outside",
+                 ticklabelmode= "period", 
+                 tickcolor= "black", 
+                 ticklen=10, 
+                 minor=dict(
+                     ticklen=4,  
+                     dtick=7*24*60*60*1000,  
+                     tick0="2016-07-03", 
```

Comment:
I changed this to sundays so that when you zoom in you don't get major ticks on sundays AND minor ticks on mondays ;)